packit_service/service/events.py

```diff
@@ -133,12 +133,12 @@ def pre_check(self) -> bool:
 
 
 class AbstractGithubEvent(Event):
-    def __init__(self, trigger: JobTriggerType, project_url=None):
+    def __init__(self, trigger: JobTriggerType, project_url: str):
         super().__init__(trigger)
         self.project_url: str = project_url
 
     def get_project(self) -> GitProject:
-        return service_config.get_project(url=self.project_url)
+        return service_config.get_project(self.project_url)
```

Comment:
Why this change? Don't you like setting arguments via key?

Reply:
reverted